tests/README.md

```diff
@@ -26,6 +26,18 @@ It is done or go without `poetry`,
 
 The second way maybe blocked the some missing dependencies at someday, so the first one is recommended.
 
+# What and how to create a unit test
+One command has a unit test, because one `git-*` command is just do one thing, so we can eat a piece of `git-*` command in one time.
+
+For example,
+
+1. The `git-alias` should has a test suite, so create `test_git_alias.py` in the directory `test`
+2. Create a test class `TestGitAlias` in the `test_git_alias.py`
+3. Create a test case `test_init`, and some test fixtures can be used, `temp_repo`, `named_temp_repo` etc.
+    * `temp_repo` is module scoped fixture which create a temporary directory and available in the test suite `test_git_alias.py`.
+    * `named_temp_repo` is just same as `temp_repo` except the custom directory renaming.
+4. Loop the third step until the 100% coverage of the function of the `git-alias`
```

Comment:
```suggestion
4. Loop the third step until the 100% coverage of the function of the `git-alias`
```

Could we have a way to measure it?

Reply:
I think the coverage of the unit test is a good way to measure it.

Reply:
The pytest coverage will be update in another PR.